pushy/src/main/java/com/eatthepath/pushy/apns/PushNotificationResponse.java

```diff
@@ -62,6 +62,16 @@ public interface PushNotificationResponse<T extends ApnsPushNotification> {
      */
     UUID getApnsId();
 
+
+    /**
+     * Returns a unique ID only available in the development environment.
+     * Useful to query push information in Push Notifications Console.
+     * @return UUID The apns unique id
+     */
+    default Optional<UUID> getApnsUniqueId() {
+        return Optional.empty();
+    }
```

Comment:
I'm not opposed to using `default` methods in general, but this would be (I believe) the first and only use in Pushy. For now, I'd recommend leaving this without a default implementation and revisiting the issue of default implementations more holistically in a separate effort/pull request.

Reply:
Ah—but then again, this will preserve backward compatibility for existing, third-party `PushNotificationResponse` implementations. Okay. I think I'm convinced.